cpp/ql/src/Security/CWE/CWE-732/FilePermissions.qll

```diff
@@ -1,5 +1,44 @@
 import cpp
-import semmle.code.cpp.commons.unix.Constants
+import semmle.code.cpp.commons.unix.Constants as UnixConstants
+
+bindingset[input]
+int parseHex(string input) {
```

Comment:
Can we have a QLDoc comment for this predicate, similarly to `parseOctal`?  (your comments are otherwise excellent)